src/models/json-api.model.ts

```diff
@@ -8,12 +8,14 @@ import { AttributeMetadata } from '../constants/symbols';
 
 export class JsonApiModel {
   id: string;
+  meta: any;
   [key: string]: any;
 
   // tslint:disable-next-line:variable-name
   constructor(private _datastore: JsonApiDatastore, data?: any) {
     if (data) {
       this.id = data.id;
+      this.meta = data.meta;
```

Comment:
Maybe a getter would be a bit nicer solution.

```
get meta(): object {
  return this.data.meta;
}
```

It would be maybe even nicer to do the same for `id` property.

@jacobbullock  What do you think?

Reply:
@safo6m I think I agree with you on this, but currently the model object isn't persisting the entire data object. It just dumps the attributes into the object.  Should we refactor the model object?

Reply:
There's no need for any refactoring (at least not regarding this topic).
Making `data` in the constructor `private` should be enough. Then you can access data as `this.data`.


```
...
 constructor(private _datastore: JsonApiDatastore, private data?: any) 
...
```

Reply:
yah, we can do it that  way.  Do  you want me to update this  PR  to include that edit for `id` you  mention above?

Reply:
Sure. Thanks.